dolt/views.py

```diff
@@ -753,6 +744,17 @@ def alter_obj(self, obj, request, url_args, url_kwargs):  # noqa: D102
         obj.reviewer = request.user
         return obj
 
+    # TODO(jathan): Figure out why we had to revert to pre-1.2.0 `get_object()`
+    # for this to work.
```

Comment:
Presumably this is due to the `kwargs["user"] = request.user` at line 757 below.